docs/example/mattermost-installation.md

```diff
@@ -169,43 +169,8 @@ To change the Mattermost installation, we need to adjust our Global policy to pr
 
 #### Instructions
 
-1. Create a GCP Service Account JSON access key:
-   
-   	1. Open [https://console.cloud.google.com](https://console.cloud.google.com) and select your project.
-   
-   	2. In the left pane, go to **IAM & Admin** and select **Service accounts**.
-   
-   	3. Click **Create service account**, name your account, and click **Create**.
-   
-   	4. Assign the `Cloud SQL Admin` role.
-   
-   	5. Click **Create key** and choose `JSON` as the key type.
-   
-   	6. Save the `JSON` file.
-   
-   	7. Click **Done**.
-
-1. Create a TypeInstance with the GCP Service Account:
-
-    ```yaml
-    # /tmp/gcp-sa-ti.yaml
-    typeInstances:
-      - alias: gcp-sa
-        typeRef:
-          path: cap.type.gcp.auth.service-account
-          revision: 0.1.0
-        attributes:
-          - path: cap.attribute.cloud.provider.gcp
-            revision: 0.1.0
-        value: { # Put here your GCP Service Account JSON.
-          "type": "service_account",
-          [...]
-        }
-    ```
-
-    ```bash
-    export TI_ID=$(capact typeinstance create -f /tmp/gcp-sa-ti.yaml -ojson | jq -r '.[].id')
-    ```
+1. Create a TypeInstance with GCP Credentials. See how to create Type Instances with [GCP credentials](typeinstances.md#gcp). When creating a new service account, assign the `Cloud SQL Admin` role to it.
```

Comment:
```suggestion
1. Create a TypeInstance with GCP Credentials. See how to create Type Instances with [GCP credentials](typeinstances.md#gcp). When creating a new service account, assign the `Cloud SQL Admin` role to it.
```


``````suggestion
1. Create a TypeInstance with GCP Service Account.

   1. Follow the [GCP Service Account TypeInstance creation](./typeinstances.md#gcp-service-account) guide to create and obtain ID of the newly created TypeInstance. 
   1. Export the TypeInstance ID as environment variable:

    ```bash
    export TI_ID={GCP Service Account TypeInstance ID}
    ```
``````